examples/global/main.rs

```diff
@@ -0,0 +1,27 @@
+#[macro_use]
+extern crate lazy_static;
+
+extern crate config;
+
+use config::Config;
+use std::error::Error;
+use std::sync::RwLock;
+
+lazy_static! {
+    static ref SETTINGS: RwLock<Config> = RwLock::new(Config::default());
+}
+
+fn try_main() -> Result<(), Box<dyn Error>> {
+    // Set property
+    #[allow(deprecated)]
+    SETTINGS.write()?.set("property", 42)?;
```

Comment:
Ah, I forgot to add that I wasn't really sure what should have gone where, so I just suppressed the warning. This should definitely be replaced with a better solution. It would be a bit weird if examples used deprecated stuff.

Reply:
Yes, it definitively is.